core-features/generate-client-and-cli-install-yarn-lockfile/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -eux
+
+VERSION=$(cat ../../.github/prisma-version.txt)
+
+cp -r ../_common/generate-client-and-cli-install/* .
+
+# global instead instead
```

Comment:
instead instead

Reply:
Improved with "install prisma globally to test that it can install the client & cli"